_build/templates/default/sass/login.scss

```diff
@@ -217,6 +217,36 @@ input[type="checkbox"] {
   }
 }
 
+#modx-login-btn {
```

Comment:
CSS rules seem absolutely identical for both buttons, so here unnecessary duplication of the code.
Should be:
```
#modx-login-btn,
#modx-fl-btn {
 ... styles here
}
```